drud/ddev
# Allow mariadb_version 5.5, 10.0/1/2/3/4 and mysql 5.5/6/7 and 8.0, fixes #1870, fixes #1627

## The Problem/Issue/Bug:

#1870 (along with #1627) requested MariaDB 10.3 and mysql versions. People have requested it for compatibility with various CMSs like Shopware.

## How this PR Solves The Problem:

* Allow MariaDB 5.5 through 10.4
* Add Oracle MySQL 5.5, 5.6, 5.7, 8.0
* Dramatic simplification of the dbserver image

## Manual Testing Instructions:

Try using `mariadb_version: 5.5` etc.
Try using `mysql_version: 8.0` etc
Try using both together (it should complain)
Try snapshots of some. And restores.

## Automated Testing Overview:

* The container tests use all these versions
* The ddevapp tests also use all these versions.

## Related Issue Link(s):

OP #1870 (mariadb 10.3)
OP #1627 (mysql variants)

## Release/Deployment notes:
